src/Components/Contacts/ImageCarret.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import Avatar from "./Avatar";
+
+const ImageCarret = ({styles, userPhoto}) => (
+    <div>
+        {userPhoto.map((photo) => (
+            <Avatar src={photo} className={styles.userImg} alt={new Date()} key ={photo} />
```

Comment:
if your component has more then 2 props please change it into multiline, and don't set space before and after "=" sign in props